packages/widgets/tests/src/menubar.spec.ts

```diff
@@ -760,6 +780,45 @@ describe('@lumino/widgets', () => {
           expect(cancelled).to.equal(true);
         });
       });
+
+      context('focus', () => {
+        it('should focus the first menu item on keyboard focus', () => {
+          let bar = createUnfocusedMenuBar();
+          bar.contentNode.focus();
+          expect(
+            bar.contentNode.contains(document.activeElement) &&
+              bar.contentNode !== document.activeElement
+          ).to.equal(true);
+          bar.dispose();
+        });
+
+        it('should loose focus on tab key', () => {
+          let bar = createUnfocusedMenuBar();
+          bar.contentNode.focus();
+          expect(
+            bar.contentNode.contains(document.activeElement) &&
+              bar.contentNode !== document.activeElement
+          ).to.equal(true);
```

Comment:
I'm not sure this test actually tests that the menu bar loses focus. Perhaps the test should be renamed, or perhaps these 4 lines should be modified and moved below line 803 (the event dispatch).

```suggestion
          expect(bar.node.contains(document.activeElement)).toBe(false); 
```

remembering the odd property that a node.contains itself, so `document.activeElement.contains(document.activeElement)` will always be true I think.